OTGTool/src/main/scala/t/manager/BatchManager.scala

```diff
@@ -568,10 +586,27 @@ class BatchManager(context: Context) {
       }
     }
 
-  def deleteRDF(title: String) = new AtomicTask[Unit]("Delete RDF data") {
+  /** Delete the RDF data associated with a batch.
+   * @param title batch ID
+   * @param metadata If not given, the entire batch is deleted.
+   *                 If given, only samples in the metadata, as well as all custom attributes in the batch will
+   *                 be deleted.
+   */
+  def deleteBatchRDF(title: String, metadata: Option[Metadata]) =
+    new AtomicTask[Unit]("Delete RDF data") {
     override def run(): Unit = {
       val bs = new BatchStore(config.triplestoreConfig)
-      bs.delete(title)
+      metadata match {
+        case Some(m) =>
+          bs.deleteSamples(title, m.samples)
+          //Delete custom attributes in the batch.
+          //If we are updating metadata, then these will be reinserted after the attribute set has been updated.
+          bs.deleteCustomAttributes(title)
+          //Also delete the old timestamp, in anticipation of a new one
+          bs.deleteTimestamp(title)
+          //Delete all data associated with the batch
+        case _ => bs.delete(title)
```

Comment:
I appreciate the comments in this method in general!
I think the indentation on this comment is off, though, if it's meant to apply to the next line